<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testWidget</class>
 <widget class="QWidget" name="testWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>testWidget</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>对方正在输入......</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="icon">
    <iconset resource="img.qrc">
     <normaloff>:/new/prefix1/img/1.png</normaloff>:/new/prefix1/img/1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>71</width>
     <height>31</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="twoPushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="threePushButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="subPushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="equalPushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bsPushButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearPushButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>c</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>300</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="img.qrc">:/new/prefix1/img/jbs.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="plusPushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>lineEdit</zorder>
  <zorder>pushButton</zorder>
  <zorder>twoPushButton</zorder>
  <zorder>threePushButton</zorder>
  <zorder>plusPushButton</zorder>
  <zorder>subPushButton</zorder>
  <zorder>equalPushButton</zorder>
  <zorder>bsPushButton</zorder>
  <zorder>clearPushButton</zorder>
  <zorder>plusPushButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
